pyanaconda/packaging/__init__.py

```diff
@@ -460,11 +460,22 @@ def _getTreeInfo(self, url, proxy_url, sslverify):
                 except URLGrabError as ex:
                     log.info("Error downloading 'treeinfo': %s", ex)
 
-                    # Don't retry if we got a 'HTTP Error 404' from both reponses
-                    # from curl (likely not to be a transient error), just move on.
-                    if "404" in str(e) and "404" in str(ex):
+                    # HTTP error
+                    if e.errno == 14 and ex.errno == 14:
```

Comment:
You could use something like
e.errno == ex.errno == 14
but it's a matter of taste.